src/VideoIntelligence/README.md

```diff
@@ -9,6 +9,16 @@
 support requests, bug reports, or development contributions should be directed to
 that project.
 
+This client supports transport over gRPC. In order to enable gRPC support please make sure to install and enable
+the gRPC extension through PECL:
+
+```sh
+$ pecl install grpc
```

Comment:
Do you think we should add notes to install gax/proto-client-php as well, since they are only suggested if you install the umbrella package?